test/index.js

```diff
@@ -19,19 +19,23 @@ test('validate-npm-package-name', function (t) {
 
   // Scoped (npm 2+)
 
-  t.deepEqual(validate('@npm/thingy'), {validForNewPackages: true, validForOldPackages: true})
+  t.deepEqual(validate('@npm/thingy'), {
+    validForNewPackages: true,
+    validForOldPackages: true,
+  })
+
```

Comment:
`@npm` is an invalid name, a test could be added for this.

Reply:
Umm, that's a loophole in the code. Ideally, the code should have some sort of function that differentiates between scope name and package name but right now it considers everything as a package name. 

But if we try to publish the package name starting with "@", it will get failed before hitting this function (In user-acl-two service).

Reply:
I wonder if we should move this check further down where also scopes are checked and then only apply it to the actual package name and not the scope. That should solve the `@` issue and make it less risky? 

https://github.com/npm/validate-npm-package-name/blob/main/index.js#L76-L88

Reply:
But I see, it's only about `@npm` being blocked in `user-acl-two`

Reply:
I've added the test case for this condition.